File: src/main/java/com/farm/service/ClassifyService.java
package com.farm.service;

import java.util.List;

import com.farm.model.Classify;
import com.farm.model.PageResult;

/**
 * 二级分类表service
 * 
 * @Date: 2019年05月14日 21:08:49 
 * @author 
 */
public interface ClassifyService extends BaseService<Classify> {
    
	List<Classify> getBycategoryId(Object id);
	
	Classify getNewsById(Classify classify);
	 
	Classify getByTreeCode(String treeCode);

    List<Classify> findALl();

	PageResult findPage(int page, int rows);

	Classify findOne(Integer id);

	void update(Classify classify);

	void deleteByIds(Integer[] ids);

	PageResult findPage(Classify classify, int page, int rows);
}

File: src/main/webapp/js/service/classifyService.js
//服务层
app.service('classifyService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../classify/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../classify/findPage.do?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../classify/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../classify/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../classify/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../classify/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../classify/search.do?page='+page+"&rows="+rows, searchEntity);
	}
	//搜索
	this.findClassifyByCategory=function(categoryId){
		return $http.post('../classify/findClassifyByCategory.do?categoryId='+categoryId);
	}
});
